sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -231,8 +261,18 @@ public String getThirdPartyHeader() {
     }
 
     final Set<String> keys = new TreeSet<>(keyValues.keySet());
```

Comment:
I know this is quite hacky, an alternative would be to let `keyValues` be a Map<String, String|Number>  aka `Map<String, Object>`